src/modules/app/containers/App.js

```diff
@@ -12,15 +12,15 @@ import PlatformProvider from '../../platform/containers/PlatformProvider'
 import routesMap from '../routesMap'
 import Switcher from './Switcher'
 import theme from '../constants/theme'
-import { Store } from 'redux'
 
 type PropsType = {
 }
 
 class App extends React.Component<PropsType> {
-  store: Store;
+  store: any;
```

Comment:
why did you need to remove the type here?

Reply:
Out of scope of this task
Store has 3 generics. There is now a warning because of the any.